src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -100,6 +106,13 @@ Tiltrotor::parameters_update()
 	param_get(_params_handles_tiltrotor.elevons_mc_lock, &l);
 	_params_tiltrotor.elevons_mc_lock = l;
 
+	/* avoid parameters which will lead to zero division in the transition code */
```

Comment:
Makes sense, if you can guarantee that these limits will never be too narrow. Else better expose them via params. These should at any rate be in the class declaration:

```
constexpr float min_front_trans dur = 0.5f;
```